README.pypi.rst

```diff
@@ -2,4 +2,40 @@
 GA4GH Client
 ============
 
-This is the GA4GH client.
+This is a client library for using the Global Alliance for Genomics and Health (GA4GH) API. This library provides an easy Python programming interface to access GA4GH compliant servers such as 1kgenome.ga4gh.org.
```

Comment:
1kgenome - > genomes. Could you make this URL active if you're going to mention it?